### ISSUE TITLE
Allow to enable/disable pretty print via CLI

### ISSUE THREAD

**Issue by robiningelbrecht** [robiningelbrecht/phpunit-pretty-print#1]
Currently when adding the extension in the PHPUnit XML file, the output is always prettified. It should be possible to enable/disable this with an argument in the CLI command

Requested by @TomasVotruba  in https://github.com/indentno/phpunit-pretty-print/pull/47

@TomasVotruba suggestion on how you would approach this?

**Comment by TomasVotruba** [comment]
Hi,

thanks for creating the isuse :) :+1:  

I don't use PHPUnit internals much, so not sure this is possible.

In PHPUnit 9 there was a "printer" option: https://github.com/indentno/phpunit-pretty-print

```bash
vendor/bin/phpunit --printer 'Sempro\PHPUnitPrettyPrinter\PrettyPrinter' tests/
```

But I can't find it in PHPUnit 10 options:

```bash
vendor/bin/phpunit -h | grep "printer"
>
```

**Comment by robiningelbrecht** [comment]
@TomasVotruba there's indeed no printer option in PHPUnit 10 anymore, everything is managed through extensions. What I'd suggest to do is something like this: 

```xml
<extensions>
    <bootstrap class="RobinIngelbrecht\PHPUnitPrettyPrint\PhpUnitExtension">
        <parameter name="enabled" value="true|false"/>
    </bootstrap>
</extensions>
```

or at runtime

```bash
> vendor/bin/phpunit -d --enable-pretty-print
```

The default value of this new option would always be "true". Your use case could then be solved by setting your XML to

```xml
<extensions>
    <bootstrap class="RobinIngelbrecht\PHPUnitPrettyPrint\PhpUnitExtension">
        <parameter name="enabled" value="false"/>
    </bootstrap>
</extensions>
```

and run

```bash
> vendor/bin/phpunit -d --enable-pretty-print
```

What do you think?

**Comment by TomasVotruba** [comment]
That seems great, I'd love to use this :+1: 

Does this work already?

**Comment by robiningelbrecht** [comment]
@TomasVotruba no, something I need to implement still. I'll try to finish this this week. Maybe tomorrow :)

**Comment by TomasVotruba** [comment]
@robiningelbrecht Awesome, thank you for helpling out :+1:

**Comment by robiningelbrecht** [comment]
@TomasVotruba, see https://github.com/robiningelbrecht/phpunit-pretty-print/pull/2, what do you think?

**Comment by robiningelbrecht** [comment]
@TomasVotruba released a new version 1.2.2

**Comment by TomasVotruba** [comment]
Thank you :clap:
